<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4" stretch="12,1,2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="1,12">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>原图</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="img">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,12">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>结果</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="img2">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="radioLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QRadioButton" name="rBtn">
        <property name="text">
         <string>Roberts</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rBtn2">
        <property name="text">
         <string>Laplacian</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rBtn3">
        <property name="text">
         <string>Canny</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rBtn4">
        <property name="text">
         <string>Roberts</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rBtn5">
        <property name="text">
         <string>Roberts</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rBtn6">
        <property name="text">
         <string>Roberts</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,1">
      <item>
       <layout class="QVBoxLayout" name="ccLayout"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="testBtn">
          <property name="text">
           <string>测试</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="saveBtn">
          <property name="text">
           <string>保存</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="saveAsBtn">
          <property name="text">
           <string>另存</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="imageBtn"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="imageBtn">
   <property name="text">
    <string>打开图片</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
